README.md

```diff
@@ -262,25 +262,36 @@ as `deps/icu` (You'll have: `deps/icu/source/...`)
 ## Current Project Team Members
 
 The io.js project team comprises a group of core collaborators and a sub-group
-that forms the _Technical Committee_ (TC) which governs the project. For more
+that forms the _Technical Steering Committee_ (TSC) which governs the project. For more
 information about the governance of the io.js project, see
 [GOVERNANCE.md](./GOVERNANCE.md).
 
-* **Isaac Z. Schlueter** ([@isaacs](https://github.com/isaacs)) &lt;i@izs.me&gt; (Technical Committee)
-* **Ben Noordhuis** ([@bnoordhuis](https://github.com/bnoordhuis)) &lt;info@bnoordhuis.nl&gt; (Technical Committee)
-* **Bert Belder** ([@piscisaureus](https://github.com/piscisaureus)) &lt;bertbelder@gmail.com&gt; (Technical Committee)
-* **Fedor Indutny** ([@indutny](https://github.com/indutny)) &lt;fedor.indutny@gmail.com&gt; (Technical Committee)
-* **Trevor Norris** ([@trevnorris](https://github.com/trevnorris)) &lt;trev.norris@gmail.com&gt; (Technical Committee)
-* **Chris Dickinson** ([@chrisdickinson](https://github.com/chrisdickinson)) &lt;christopher.s.dickinson@gmail.com&gt; (Technical Committee)
+### TSC (Technical Steering Committee)
+
+* **Ben Noordhuis** ([@bnoordhuis](https://github.com/bnoordhuis)) &lt;info@bnoordhuis.nl&gt; (TSC)
+* **Bert Belder** ([@piscisaureus](https://github.com/piscisaureus)) &lt;bertbelder@gmail.com&gt; (TSC)
+* **Fedor Indutny** ([@indutny](https://github.com/indutny)) &lt;fedor.indutny@gmail.com&gt; (TSC)
+* **Trevor Norris** ([@trevnorris](https://github.com/trevnorris)) &lt;trev.norris@gmail.com&gt; (TSC)
+* **Colin Ihrig** ([@cjihrig](https://github.com/cjihrig)) &lt;cjihrig@gmail.com&gt; (TSC)
+* **Alexis Campailla** ([orangemocha](https://github.com/orangemocha)) &lt;v-alcamp@microsoft.com&gt; (TSC)
+* **James M Snell** ([jasnell](https://github.com/jasnell)) &lt;jasnell@gmail.com&gt; (TSC)
+* **Julien Gilli** ([misterdjules](https://github.com/misterdjules)) &lt;jgilli@nodejs.org&gt; (TSC)
+* **Michael Dawson** ([mhdawson](https://github.com/mhdawson)) &lt;michael_dawson@ca.ibm.com&gt; (TSC)
+* **Steven R Loomis** ([srl295](https://github.com/srl295)) &lt;srloomis@us.ibm.com&gt; (TSC)
```

Comment:
afaik Steven said he didn't need to be here since that would put IBM over the limit?